### PR TITLE
Feature - email notification calculation method change [#462]

### LINKED ISSUE #462: Notification email if calculation method changed for PD with a previously submitted+accepted progress report
Alert if partner changes the calculation method in PRP for indicators if any report for that PD has already accepted by UNICEF. Email should contain a link to the calculation method tab in question for the pd which the user can click on. 

Please put a base email template in place if one doesn't already exist that all such emails will use. 

------------------------

Email to: Focal Point(s) associated with the PD
Subject Line: Calculation Method for indicators in PD $name$ have changed
Dear <UNICEF Focal Point Name>,
This is to inform you that the Calculation Method for indicators in PD $name$ have changed.
-$Partner Name$

### PR DESCRIPTION
**Pull request by n1207n**
##### Feature list
* _Describe the list of features from Django_
* Email template files for notifying UNICEF focal points if calculation method changes in Programme Document that has accepted ProgressReports
* Email utility function to call send_email()
* Email notification boolean flag logic in CalculationMethodAPIView post method

* _Describe the list of features from Polymer_

##### Progress checker
- [x] Django

- [ ] Polymer

- [ ] Django test cases

- [ ] Polymer test cases

### CODE REVIEW

**Comment by chachra** [comment]
@maciej-jaworski if you can review tomorrow please. assigned you.

@n1207n - minor code conflict. tx.

File: django_api/django_api/settings/base.py [hunk 1]
```diff
@@ -60,6 +60,7 @@
 EMAIL_HOST_USER = os.getenv('EMAIL_HOST_USER')
 EMAIL_HOST_PASSWORD = os.getenv('EMAIL_HOST_PASSWORD')
 EMAIL_USE_TLS = True
+EMAIL_FROM_ADDRESS = 'no-reply@etools.unicef.org'
```
Comment (maciej-jaworski) [inline django_api/django_api/settings/base.py]: Shouldn't we make this customisable through an env var?
Also any reason we're not using `DEFAULT_FROM_EMAIL`,  which saves us from passing that email around.

https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-DEFAULT_FROM_EMAIL

Comment (n1207n) [reply django_api/django_api/settings/base.py]: Yeah I forgot about this settings value ahaha. Will implement.

**Comment by n1207n** [comment]
DEFAULT_FROM_EMAIL settings is now used in settings file and send_email_from_template invocation. Feel free to merge in.

**Pull request merged**